### PR TITLE
Add WWW-AdventCalendar as a submodule

### PR DESCRIPTION
**Pull request by oalders**
- Build site using 5.38 rather than 5.36
- Add WWW-AdventCalendar as a submodule

### CODE REVIEW

File: inc/WWW-AdventCalendar
Comment (thibaultduponchelle) [inline inc/WWW-AdventCalendar:1]: I think you should at least target v1.113 (because of the `--https` option)?

Comment (oalders) [reply inc/WWW-AdventCalendar:1]: Yeah, I'll have to ensure it's using the correct branch.

Comment (oalders) [reply inc/WWW-AdventCalendar:1]: Actually, with `perladvent` now being the default branch, I think we are all set here?

Comment (thibaultduponchelle) [reply inc/WWW-AdventCalendar:1]: It should be good now

**Comment by oalders** [comment]
Let's see if this works!

**Pull request merged**